src/frontends/common/include/openvino/frontend/extension/op.hpp

```diff
@@ -46,6 +46,12 @@ class OpExtensionBase : public BaseConversionType {
     OpExtensionBase(const std::string& fw_type_name,
                     const std::map<std::string, std::string>& attr_names_map = {},
                     const std::map<std::string, ov::Any>& attr_values_map = {});
+
+    // Maps op with a given type and domain in FW and OV type given in template parameter
+    OpExtensionBase(const std::string& fw_type_name,
```

Comment:
Why do we add this constructor to `BaseOpExtension`?
Should it be inside `onnx::OpExtension`?

Reply:
good point, we definitely need to move thic ctor to `onnx::OpExtension`

Reply:
@ilyachur @itikhono done